lib/Echidna/SymExec/Common.hs

```diff
@@ -224,26 +147,30 @@ getUnknownLogs = mapMaybe (\case
 exploreMethod :: (MonadUnliftIO m, ReadConfig m, TTY m) =>
   Method -> SolcContract -> EVM.Types.VM Concrete RealWorld -> Addr -> EConfig -> VeriOpts -> SolverGroup -> Fetch.RpcInfo -> IORef ContractCache -> IORef SlotCache -> m ([TxOrError], PartialsLogs)
   
-exploreMethod method contract vm defaultSender conf veriOpts solvers rpcInfo contractCacheRef slotCacheRef = do
-  --liftIO $ putStrLn ("Exploring: " ++ T.unpack method.methodSignature)
-  --pushWorkerEvent undefined
-  calldataSym@(cd, constraints) <- mkCalldata (Just (Sig method.methodSignature (snd <$> method.inputs))) []
+exploreMethod method _ vm defaultSender conf veriOpts solvers rpcInfo contractCacheRef slotCacheRef = do
+  calldataSym@(_, constraints) <- mkCalldata (Just (Sig method.methodSignature (snd <$> method.inputs))) []
+  let
+    cd = fst calldataSym
   let
     fetcher = cachedOracle contractCacheRef slotCacheRef solvers rpcInfo
     dst = conf.solConf.contractAddr
-    vmSym = abstractVM calldataSym contract.runtimeCode Nothing False
-  vmSym' <- liftIO $ stToIO vmSym
   vmReset <- liftIO $ snd <$> runStateT (fromEVM resetState) vm
-  let vm' = vmReset & execState (loadContract (LitAddr dst))
-                    & vmMakeSymbolic conf.txConf.maxTimeDelay conf.txConf.maxBlockDelay
-                    & #constraints %~ (++ constraints ++ (senderConstraints conf.solConf.sender))
-                    & #state % #callvalue .~ TxValue
-                    & #state % #caller .~ SymAddr "caller"
-                    & #state % #calldata .~ cd
-                    & #env % #contracts .~ (Map.union vmSym'.env.contracts vm.env.contracts)
+  let
+    vm' = vmReset & execState (loadContract (LitAddr dst))
+                  & #tx % #isCreate .~ False
+                  & #state % #callvalue .~ TxValue
+                  & #state % #caller .~ SymAddr "caller"
+                  & #state % #calldata .~ cd
+                  -- & #env % #contracts .~ (Map.union vmSym'.env.contracts vm.env.contracts)
```

Comment:
Why are contracts not loaded on the symbolic VM?

Reply:
Removed this comment. All the tests seem to be working fine, even with multiple contracts. If we found some contract is not loaded, we can restore this line.